benchmark/scenarios/no_load.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+No load scenario for the control service benchmarks.
+"""
+
+from zope.interface import implementer
+
+from twisted.internet.defer import Deferred, succeed
+
+from .._interfaces import IScenario
+
+
+@implementer(IScenario)
+class NoLoadScenario(object):
+    """
+    A scenario that places no additional load on the cluster.
+    """
+
+    def __init__(self, clock, control_service):
+        self._maintained = Deferred()
+
+    def start(self):
+        """
+        :return: A Deferred that fires when the desired scenario is
+            established (e.g. that a certain load is being applied).
+        """
+        return succeed(None)  # no setup needed
+
+    def maintained(self):
+        """
+        :return: A Deferred that fires with an errback if the desired
+            scenario fails to hold between being established and being
+            stopped.  This Deferred never fires with a callback.
+        """
+        return self._maintained
```

Comment:
For real scenarios, there will be a monitoring "thread" that will fire the Deferred, and the client that adds callbacks.  These two users of the Deferred need to get the same Deferred.  It doesn't matter here so much, but it also doesn't seem a big problem.